testing/constraints-3.9.txt

```diff
@@ -21,7 +21,7 @@ opentelemetry-api==1.1.0
 opentelemetry-instrumentation==0.20b0
 opentelemetry-sdk==1.1.0
 packaging==24.2.0
-pandas==1.1.4
```

Comment:
I confirmed that `pandas==1.2.5` reproduces

```
FAILED tests/unit/test_table.py::TestRowIterator::test_to_dataframe_w_bqstorage_no_streams - ValueError: need at least one array to concatenate
```

and `pandas==1.3.0`  can pass

Reply:
Thanks for updating the constraint file!